hi-ml-histopathology/src/histopathology/models/deepmil.py

```diff
@@ -155,7 +113,8 @@ def get_metrics(self) -> nn.ModuleDict:
                                   MetricsKey.CONF_MATRIX: ConfusionMatrix(num_classes=2, threshold=threshold)})
 
     def log_metrics(self,
-                    stage: str) -> None:
+                    stage: str,
+                    pl_module) -> None:
```

Comment:
docu for the arguments missing. Also, pl_module is a too generic name given that self is also a Module.